docker.sh

```diff
@@ -82,13 +82,21 @@ for version in 136; do
     ${DEBUG} docker build \
         --build-arg MODEL_VERSION="${version}" \
         --tag pecan/model-sipnet-${version}:latest \
-        --file models/sipnet/Dockerfile .
+        models/sipnet
 done
 
 # build ed2
 for version in git; do
     ${DEBUG} docker build \
         --build-arg MODEL_VERSION="${version}" \
         --tag pecan/model-ed2-${version}:latest \
-        --file models/ed/Dockerfile .
+        models/ed
+done
+
+# build maespa
+for version in git; do
+    ${DEBUG} docker build \
+        --build-arg MODEL_VERSION="${version}" \
+        --tag pecan/model-maespa-${version}:latest \
```

Comment:
Again, this `latest` should be configurable. This could use a separate variable (e.g. `MODEL_VERSION`).

Reply:
using IMAGE_VERSION to keep it in sync with the pecan version number.